taxonomies/ingredients_processing.txt

```diff
@@ -464,20 +467,51 @@ de:angereichert
 fr:enrichi, enrichie, enrichis
 nl:verrijkt
 
+#<en:unadded
 en:unenriched
 
 en:bromated
 es:bromado
 
+#<en:unadded
 en:unbromated, un-bromated
 
+#en:comment:salt has been added to the ingredient
+#<en:added
+en:salted
 de:gesalzen, gesalzene, gesalzenes, gesalzener
+es:salado, salada, salados, saladas
+# es:comment:careful for es:ensalada
+fr:salé, salée, salés, salées, sales, salees
```

Comment:
for French (and Spanish, Italian, Portuguese and English) de-accented versions are generated automatically